VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -271,8 +267,22 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
         return true;
     }
 
-    handleParticipantStatusChange(message: ParticipantStatusMessage): void {
+    private newParticipantEnteredHandshake(newParticipantEntered) {
+        this.logger.info(`${this.loggerPrefix} Waiting 3 seconds before sending handshake`);
+        if (this.participant.hearing_role !== HearingRole.JUDGE && this.participant.hearing_role !== HearingRole.STAFF_MEMBER) {
+            setTimeout(() => {
+                this.logger.info(`${this.loggerPrefix} Sending handshake for entry of: ${newParticipantEntered}`);
+                this.publishMediaDeviceStatus();
+                this.eventService.publishParticipantHandRaisedStatus(this.conferenceId, this.participant.id, this.handRaised);
+            }, 3000); // 3Seconds: Give 2nd host time initialise participants, before receiving status updates
```

Comment:
Not sure the 3 seconds delay will affect the syncing issue between the Judge and SM syncing and reflecting in the UI. Lets test and see if all OK.